lib/phoenix_storybook/templates/layout/live.html.heex

```diff
@@ -7,144 +7,133 @@
       </.link>
     </div>
 
-    <%= if (themes = themes(@socket)) || color_mode?(@socket) do %>
```

Comment:
nit, but switched to encapsulating div for less tags and increased readability, can revert if desired